packages/app-tests/e2e/templates/create-document-from-template.spec.ts

```
@@ -1,7 +1,11 @@
 import { expect, test } from '@playwright/test';
+import fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)